src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -124,6 +124,8 @@ private interface BuiltInProfile {
         /*
          * Deferral is only used for standard profiles. Enabling the appropriate
          * access privileges is handled at a lower level.
```

Comment:
I wonder if “Enabling the appropriate access privileges is handled at a lower level” is still relevant after `SecurityManager` was removed.

Reply:
The first sentence can also be removed. The newly added sentence repeats the first one.